packages/next-swc/crates/next-dev-tests/tests/integration.rs

```diff
@@ -339,16 +344,6 @@ async fn run_test(resource: PathBuf) -> JsResult {
     });
     tt.wait_task_completion(task, true).await.unwrap();
 
-    // This sometimes fails for the following test:
-    // test_tests__integration__next__webpack_loaders__no_options__input
-    retry(
-        (),
-        |()| std::fs::remove_dir_all(&resource_temp),
-        3,
-        Duration::from_millis(100),
-    )
-    .expect("failed to remove temporary directory");
-
```

Comment:
I'm guessing you removed this because we're cleaning that directory when running tests again anyway? I still think it's good to clean up after running tests.